### PR TITLE
Pagopa 918 gestione flag all ccp

### PR DESCRIPTION
**Pull request by FedericoRuzzier**
#### List of Changes
Adding boolean flag for the exclusion of the Poste bundles in the getFees API

#### Motivation and Context
This addition was required to perform an additional filter operation on the API

#### How Has This Been Tested?
This flag has been tested manually and through Junit tests

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/main/java/it/gov/pagopa/afm/calculator/controller/CalculatorController.java [hunk 1]
```diff
@@ -87,7 +87,8 @@ public List<Transfer> getFeesByPsp(
       @Parameter(description = "PSP identifier", required = true) @PathVariable("idPsp")
           String idPsp,
       @RequestBody @Valid PaymentOptionByPsp paymentOptionByPsp,
-      @RequestParam(required = false, defaultValue = "10") Integer maxOccurrences) {
+      @RequestParam(required = false, defaultValue = "10") Integer maxOccurrences,
```
Comment (jacopocarlini) [inline src/main/java/it/gov/pagopa/afm/calculator/controller/CalculatorController.java:90]: Can you provide specific swagger descriptions?

**Review by jacopocarlini — CHANGES_REQUESTED** [review]
It's OK, just provide a description for the swagger

**Comment by jacopocarlini** [comment]
remember to update the swagger

**Comment by FedericoRuzzier** [comment]
update_code

File: helm/values-prod.yaml [hunk 2]
```diff
@@ -69,6 +69,7 @@ microservice-chart:
     COSMOS_URI: https://pagopa-p-weu-afm-marketplace-cosmos-account.documents.azure.com:443/
     ISSUER_RANGE_TABLE: 'pagopapweuafmsaissuerrangetable'
     AMOUNT_THRESHOLD: '0'
+    ID_PSP_POSTE: 'BPPIITRRXXX'
```
Comment (jacopocarlini) [inline helm/values-prod.yaml:72]: is this the right value?

File: src/main/java/it/gov/pagopa/afm/calculator/repository/CosmosRepository.java [hunk 3]
```diff
@@ -36,8 +36,8 @@ public class CosmosRepository {
 
   @Autowired UtilityComponent utilityComponent;
 
-  @Value("${pspPoste.id}")
-  private String pspPosteId;
+  @Value("#{'${pspPoste.id}'.split(',')}")
+  private List<String> pspPosteId;
```
Comment (jacopocarlini) [inline src/main/java/it/gov/pagopa/afm/calculator/repository/CosmosRepository.java:40]: why use a list?

Comment (FedericoRuzzier) [reply src/main/java/it/gov/pagopa/afm/calculator/repository/CosmosRepository.java:40]: Criteria method NOT_IN requests a list, plus could be useful if other psp are added in future

File: helm/values-prod.yaml (hunk 2 above)
Comment (FedericoRuzzier) [reply helm/values-prod.yaml:72]: I took it from API Config in PROD

File: openapi/openapi-node.json [hunk 4]
```diff
@@ -8,7 +8,7 @@
   },
   "servers": [
     {
-      "url": "http://localhost:8080",
+      "url": "http://localhost:8586",
```
Comment (jacopocarlini) [inline openapi/openapi-node.json:11]:
```suggestion
      "url": "http://localhost:8080",
```

File: openapi/openapi.json [hunk 5]
```diff
@@ -8,7 +8,7 @@
   },
   "servers": [
     {
-      "url": "http://localhost:8080",
+      "url": "http://localhost:8586",
```
Comment (jacopocarlini) [inline openapi/openapi.json:11]:
```suggestion
      "url": "http://localhost:8080",
```

**Review by jacopocarlini — APPROVED** [review]
Godd work! 👍

**Pull request merged**